README.md

```diff
@@ -1,106 +1,170 @@
+In Conjur (both OSS and DAP), environment variables are mapped to configuration variables
+by prepending `CONJUR_` to the all-caps name of the configuration variable. For example,
+`appliance_url` is `CONJUR_APPLIANCE_URL`, `account` is `CONJUR_ACCOUNT` etc.
 
-`CONJUR_AUTHN_LOGIN` - user/host identity
+_NOTE:_ For ways to set credentials (for `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY`
+environment variables ONLY) in the app instead of environment variables, see the
+[Authorization Examples](#authorization-examples) and/or [system properties](#system-properties)
+section below.
 
-`CONJUR_AUTHN_API_KEY` - user/host API key
+The following environment variables need to be included in the apps runtime environment in
+order use the Conjur API:
 
-`CONJUR_AUTHN_URL` - (optional) Set this variable if using alternative authentication method
+- `CONJUR_APPLIANCE_URL` - Conjur endpoint (OSS/DAP) in URL format.
+- `CONJUR_ACCOUNT` - Conjur account that you are connecting to. This value is set during Conjur deployment.
+- `CONJUR_AUTHN_LOGIN` - User/host identity
+- `CONJUR_AUTHN_API_KEY` - User/host API key
+- `CONJUR_AUTHN_URL` - (optional) Alternate authn endpoint. Default endpoint uses `<applianceUrl>/authn`
+  for generic username and password/API key login flow.
 
-For example, specify the environment variables like so:
+For example, you can specify the environment variables like so:
 
-```bash
-CONJUR_ACCOUNT=myorg
-CONJUR_AUTHN_LOGIN=host/myhost.example.com
-CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
-CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api
+```sh-session
+export CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api
+export CONJUR_ACCOUNT=myorg
+export CONJUR_AUTHN_LOGIN=host/myhost.example.com
+export CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s
 ```
 
-In order to set `CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY`, you will need to add the host to Conjur with the proper privileges in policy. 
+or you could provide these at runtime to your jar:
+```sh-session
+$ CONJUR_APPLIANCE_URL=https://conjur.myorg.com/api \
+  CONJUR_ACCOUNT=myorg \
+  CONJUR_AUTHN_LOGIN=host/myhost.example.com \
+  CONJUR_AUTHN_API_KEY=sb0ncv1yj9c4w2e9pb1a2s \
+  java -jar myConjurClient.jar
+```
+If you are using a host-based user like this example shows, you will need to add the host
+to Conjur with the proper privileges in policy in order to know the appropriate
+`CONJUR_AUTHN_LOGIN` and `CONJUR_AUTHN_API_KEY` values.
 
-For this _specific_ example, we will define the Java app as a host and grant permissions over some variable. This variable will later hold a secret and we will 
-show how to retrieve this secret using the Conjur Java API once a connection is established. If you have a different use case, you can use the below policy as the basis for 
-another desired outcome. If a policy already exists for your host, you can skip the next set of steps.
+For this _specific_ example, we will define the Java app as a host and grant permissions over
```

Comment:
this example seems really out of place here. I see two options for it.
- does it add any value that makes it worth keeping? we could move it to a section with the other examples below (maybe before them?) as "Example workflow for setting up Conjur"
- can we scrap it altogether and just link out to our docs for more info on setting up and configuring conjur and loading policy?

Reply:
You make a good point - this whole example can be scrapped so I'll do that.